user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -161,14 +161,19 @@ export class RegistrationFormComponent implements OnInit {
 
   ngOnInit() {
     this.route.data.subscribe(({ registration }) => {
-      this.registrationForm = registration;
-      this.registrationForm.firstHackathon = false;
-      this.registrationForm.eighteenBeforeEvent = null;
-      this.registrationForm.mlhcoc = null;
-      this.registrationForm.mlhdcp = null;
-      this.parsePhone(this.registrationForm.phone);
-      this.phoneNoUse = this.prettifiedPhone;
-      this.diet_restr = this.registrationForm.dietaryRestriction !== null;
+      this.registrationForm = new Registration();
+      //Data to keep
+      //TODO: make this into an actual function to be called
+      this.registrationForm.firstName = registration.firstName;
+      this.registrationForm.lastName = registration.lastName;
+      this.registrationForm.ethnicity = registration.ethnicity;
+      this.registrationForm.gender = registration.gender;
+      this.registrationForm.allergies = registration.allergies;
+      // this.registrationForm.major = registration.major;
```

Comment:
Why commented?

Reply:
In the form, the saved answer came up but underneath was the "enter your university" text. It's just a minor gross UI bug, but I'd like to have this fix out to stop the error we saw and then go back and figure out how to replace the text when I have more time